Nim.cs

```diff
@@ -23,14 +30,51 @@ public Nim()
         }
         public void StartGame()
         {
-            Console.WriteLine("Hello! Let's play a game of Nim (21)!");
+            Console.WriteLine("Hello! Let's play a game of Nim!");
+            switch (GetGameType()) {
+                case 21:
+                    Nim21();
+                    break;
+                case 100:
+                    Nim100();
+                    break;
+            }
+
+            GameLoop();
+        }
+
+        public void Nim21()
+        {
+            gameType = Variant.Nim21;
             Console.WriteLine("The game begins with one player saying the number 1, then each player takes turns saying 1, 2, or 3, adding to a running total.");
             Console.WriteLine("The player who is forced to say a number resulting in a total of 21 or greater loses!");
             Console.WriteLine("I'll go first. 1, for a total of 1.");
-            GameLoop();
         }
 
-        private static int GetChoice(int total)
+        public void Nim100()
+        {
+            gameType = Variant.Nim100;
+            Console.WriteLine("The game begins with one player saying a number between 1 and 10 (inclusive), then each player takes turns saying a number between 1 and 10 (inclusive), adding to a running total.");
+            Console.WriteLine("The player who says 100 or greater first wins!");
+            var random = new Random();
+            var choice = random.Next(1, 11);
+            Total = choice;
+            Console.WriteLine("I'll go first. {0}, for a total of {1}.", choice, Total);
+
+        }
+
+        private static int GetGameType()
```

Comment:
A property called `GameType` and a function called `GetGameType` that do different things are confusing.  How about renaming this to `PromptForGameType` or something like that?